test/e2e/mattermost_test.go

```diff
@@ -133,6 +133,13 @@ func mattermostScaleTest(t *testing.T, f *framework.Framework, ctx *framework.Te
 	err = f.Client.Update(context.TODO(), exampleMattermost)
 	require.NoError(t, err)
 
+	// wait for test-mm to reach 2 replicas
```

Comment:
nit: wait for test-mm to reach 1 replica

Reply:
:(

Reply:
fixed